src/app/books.ts

```diff
@@ -2433,12 +2421,208 @@ export const BOOKS : Book[] = [
+		]
+	}
+	,{
+		"id": 85,
+		"title": "Off The Mark",
+		"edition": "1",
+		"img": "https://cdn-img.fimfiction.net/story/ckkw-1534612975-415985-medium",
+		"rating": "T",
+		"dateAdded": "2019/04/24",
+		"expiry": null,
+		"authors":[
+			{
+				"name": "Goldfur",
+				"url": "https://www.fimfiction.net/user/84240/Goldfur"
+			}
+		],
+		"links": [
+			{
+				"url": "https://www.fimfiction.net/story/415985/off-the-mark",
+				"title": "Fimfiction.net"
+			},
+			{
+				"url": "https://www.amazon.com/dp/1797620347",
+				"title": "Perfect-Bound Paperback"
+			}
+		],
+		"tags": [
+			"fimfic", "paperback"
+		]
+	}
+	,{
+		"id": 86,
+		"title": "Background Pony",
+		"edition": "3",
+		"img": "https://cdn-img.fimfiction.net/story/zhml-1432427782-19198-medium",
+		"rating": "T",
+		"dateAdded": "2019/04/24",
+		"expiry": null,
+		"authors":[
+			{
+				"name": "shortskirtsandexplosions",
+				"url": "https://www.fimfiction.net/user/1491/shortskirtsandexplosions"
+			}
+		],
+		"links": [
+			{
+				"url": "https://www.fimfiction.net/story/19198/Background-Pony",
+				"title": "Fimfiction.net"
+			},
+			{
+				"url": "https://forms.gle/JRbnp3q5Nnp3kmfXA",
+				"title": "Signup Form"
+			}
+		],
+		"tags": [
+			"fimfic", "hardcover", "signup"
+		]
+	}
+	,{
+		"id": 87,
+		"title": "Hard Reset",
+		"edition": "3",
+		"img": "https://cdn-img.fimfiction.net/story/gmtr-1432458620-67362-medium",
+		"rating": "T",
+		"dateAdded": "2019/04/24",
+		"expiry": null,
+		"authors":[
+			{
+				"name": "Eakin",
+				"url": "https://www.fimfiction.net/user/15048/Eakin"
+			}
+		],
+		"links": [
+			{
+				"url": "https://www.fimfiction.net/story/67362/hard-reset",
+				"title": "Fimfiction.net"
+			},
+			{
+				"url": "https://forms.gle/JRbnp3q5Nnp3kmfXA",
+				"title": "Signup Form"
+			}
+		],
+		"tags": [
+			"fimfic", "hardcover", "signup"
+		]
+	}
+	,{
+		"id": 88,
+		"title": "Stardust",
+		"edition": "2",
+		"img": "https://cdn-img.fimfiction.net/story/ddv4-1432482453-100455-medium",
+		"rating": "T",
+		"dateAdded": "2019/04/24",
+		"expiry": null,
+		"authors":[
+			{
+				"name": "Arad",
+				"url": "https://www.fimfiction.net/user/45831/Arad"
+			}
+		],
+		"links": [
+			{
+				"url": "https://www.fimfiction.net/story/100455/stardust",
+				"title": "Fimfiction.net"
+			},
+			{
+				"url": "https://forms.gle/JRbnp3q5Nnp3kmfXA",
+				"title": "Signup Form"
+			}
+		],
+		"tags": [
+			"fimfic", "hardcover", "signup"
+		]
+	}
+	,{
+		"id": 89,
+		"title": "Tales of the Sun",
+		"edition": "1",
+		"img": "https://static.wixstatic.com/media/45d896_4da7b378c7b049479da6e5f7cc74bacf~mv2_d_4335_2941_s_4_2.jpg/v1/fill/w_1125,h_717,al_c,q_85,usm_0.66_1.00_0.01/45d896_4da7b378c7b049479da6e5f7cc74bacf~mv2_d_4335_2941_s_4_2.webp",
```

Comment:
I've always had a preferance for using the fimfic link, but for anthologies... well, I'm still not sure. This doesn't even have the title on it, though clearly this is a book cover, front and back.

Reply:
It's the MoI link, I was originally going to crop out just the front side so it would be in portrait and more fitting with the site, but you mentioned you didn't want to host anything.  I am still unsure if we should link the cover of one of the stories in the anthology or the cover/image representing them and we should have further discussion over this.